enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+
+    ```go
+    `mkdir -p {{ .FBCDir }} && \
+    echo '{{ .FBCContent }}' >> {{ .FBCFile }} && \
+    opm serve {{ .FBCDir }} -p {{ .GRPCPort }}`
+    ```
+
+    - Note: It is worth noting that the new command to serve a File-Based Catalog to GRPC has been changed from `opm registry serve` to `opm serve`. The above command template reflects this change. `opm registry` commands will be deprecated in a future release.
+
+10. In this way, the File-Based Catalog that was generated by the Operator SDK will be served to the gRPC API via the specified port (defaulted to `50051`) without any user-facing changes.
+
+11. The remaining resources (`CatalogSource`, `Subscription`, `OperatorGroup`) are created in the same way as before. The run bundle command does not change from here on.
+
+- Note: If a SQLite index is passed, the old commands will be used in the registry pod and none of the FBC creation will happen. Below is the container creation command used to handle SQLite index images (the optional flags are skipped here for simplicity purposes):
+
+    ```go
+    `mkdir -p {{ dirname .DBPath }} && \
+    {{- range $i, $item := .BundleItems }}
+    opm registry add -d {{ $.DBPath }} -b {{ $item.ImageTag }} --mode={{ $item.AddMode }} && \
+    {{- end }}
+    opm registry serve -d {{ .DBPath }} -p {{ .GRPCPort }}`
+    ```
+
+#### Example UX Scenarios
+
+Running the following scenarios results in different outcomes for FBC creation. These are some example uses of the `run bundle` command. This is a not an exhaustive list of the command with the other flags.
+
+```sh
+- operator-sdk run bundle <bundle-image>
+- operator-sdk run bundle <bundle-image> --index-image quay.io/example/testFBCCatalog
+- operator-sdk run bundle <bundle-image> --index-image quay.io/example/testSQLiteCatalog
+```
+
+- `operator-sdk run bundle <bundle-image>` would create a File-Based Catalog (FBC) on the fly by rendering the bundle image and using the default upstream opm image `quay.io/operator-framework/opm:latest`. Bundle image contents will be served over a gRPC port through the ephemeral registry pod.
+- `operator-sdk run bundle <bundle-image> --index-image quay.io/example/testFBCCatalog` would ensure that the bundle image doesn't exist in the `testFBCCatalog` index. If it exists, then we error out. If not, we add the bundle image to the index image by rendering both of these images using the Operator Registry APIs thereby appending the bundle contents to the index and serving the index over the gRPC port.
+- `operator-sdk run bundle <bundle-image> --index-image quay.io/example/testSQLiteCatalog` will still be supported in the old way by making use of `opm registry add` and `opm registry serve` commands under the hood and without creating an FBC or rendering the SQLite based image.
+
+#### Prototype Links
+
+- [PR](<https://github.com/operator-framework/operator-sdk/pull/5671>) that implements FBC support in run bundle.
```

Comment:
Is not it the actual implementation?

Reply:
The plan is to have run bundle go into a feature branch on the SDK repository. Then, we will also open a PR for bundle-upgrade related changes to the same feature branch. After both the PRs get merged, we will open a PR from the feature branch to the master branch. We are doing this so both run bundle and run bundle-upgrade related changes go into master together.

Reply:
This is the actual implementation PR for `run bundle` working with both FBC and SQLite scenarios.

Reply:
My point is that is not the Prototype as described in the doc.
Also, I do see a section Prototype in the template. 
I think it would be under Implementation History

Reply:
Moved it to Implementation history.